xla/service/gpu/transforms/gemm_rewriter_test.cc

```diff
@@ -5032,6 +5032,60 @@ TEST_P(ParameterizedFp8GemmRewriteTest, UnscaledABUnscaledDMatrixBiasF8) {
       )");
 }
 
+TEST_P(ParameterizedFp8GemmRewriteTest, ScaledABUnscaledDColMajorLhsF8) {
+  const char* hlo_text = R"(
```

Comment:
Can you change this test to involve batch dimensions as well? Just so that we can see that those can be handled as well.

Reply:
Updated.